cmd/worker/clickhouse/clickhouse.go

```diff
@@ -77,36 +77,34 @@ var migrations embed.FS
 // ApplyMigrations applies all previously unapplied migrations. If successful,
 // it returns the number of migrations previously applied (old) and the number
 // of newly applied migrations.
-func (c *Client) ApplyMigrations(ctx context.Context) (old int, applied int, _ error) {
+func (c *Client) ApplyMigrations(ctx context.Context) (applied int, _ error) {
```

Comment:
Please update the function doc. (Or just remove it, it's kinda obvious with one return variable.)

Reply:
Changed